azure-pipelines.yml

```diff
@@ -54,6 +54,6 @@ steps:
   - script: |
       source /opt/ros/foxy/setup.bash
       cd ~/ros2_rust_ws
-      colcon test --packages-select rclrs rclrs_common rclrs_examples rosidl_generator_rs
+      colcon test --packages-select rclrs rclrs_examples rosidl_generator_rs
```

Comment:
I've submitted https://github.com/ros2-rust/ros2_rust/pull/74 for review to remove the Azure pipelines config, now that we're using GitHub actions for CI. Can you approve that PR and rebase your branch? Thanks.

Reply:
Done.